src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,86 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SPLIT_DELIMITER_SCV = ",";
+    public static final String DATA_FIRST = "supply";
+    public static final String DATA_SECOND = "buy";
+    public static final String DATA_THIRD = "result";
+    public static final int READ_FILE_DATA_INDEX_ONE = 0;
+    public static final int READ_FILE_DATA_INDEX_TWO = 1;
+
+    public String[][] readeFile(String fileName) {
+        long counterReadLines = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String readOneLine = bufferedReader.readLine();
+            while (readOneLine != null) {
+                counterReadLines++;
+                readOneLine = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read data from: " + fileName, e);
+        }
+        String[][] readAllLines = new String[(int) counterReadLines][];
+        int arrayIndex = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            String readOneLine = bufferedReader.readLine();
+            while (readOneLine != null) {
+                String[] arrayToSplit = readOneLine.split(SPLIT_DELIMITER_SCV);
+                readAllLines[arrayIndex] = arrayToSplit;
+                arrayIndex++;
+                readOneLine = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can not read data from: " + fileName, e);
+        }
+        return readAllLines;
```

Comment:
Your method called read from file, but here make some parsing, plz read from file and return it in String type